src/python/grpcio/grpc/_cython/_cygrpc/records.pyx.pxi

```diff
@@ -319,14 +339,16 @@ cdef class ChannelArgs:
       if not isinstance(arg, ChannelArg):
         raise TypeError("expected list of ChannelArg")
     self.c_args.arguments_length = len(self.args)
-    self.c_args.arguments = <grpc_arg *>gpr_malloc(
-        self.c_args.arguments_length*sizeof(grpc_arg)
-    )
+    with nogil:
+      self.c_args.arguments = <grpc_arg *>gpr_malloc(
+          self.c_args.arguments_length*sizeof(grpc_arg)
+      )
```

Comment:
Move this to the end of the previous line?

There's not yet a cyformat to match pyformat, right? :-P